<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="vehicle" default="X1" />
  <arg name="type" default="robot" />
  <arg name="sharePoseGraph" default="true" />
  <!-- No need to send images in virtual unless testing -->
  <arg name="sendImages" default="false" />
  <!-- If ack's are reliable, set this larger.  If a lot are dropped, set small. -->
  <arg name="fuseDist" default="3.0" />
  <arg name="singleGoalDeconflict" default="true" />
  <arg name="myBeacons" default="" />
  <arg name="potentialNeighbors" default="" />
  <arg name="stopCheck" default="15" />
  <arg name="stopTopic" default="estop" />
  <arg name="stopCommand" default="true" />
  <arg name="turnDetect" default="false" />
  <arg name="delayDrop" default="false" />
  <arg name="reverseDrop" default="true" />
  <arg name="mappingRange" default="5.0" />
  <arg name="useExtTraj" default="false" />

  <include file="$(find bobcat)/launch/bobcat.launch">
    <arg name="maxBeacons" value="12" />
    <arg name="totalBeacons" value="36" />
    <arg name="simcomms" value="false" />
    <arg name="anchorDropDist" value="80" />
    <arg name="dropDist" value="50" />
    <arg name="useVirtual" value="true" />
    <arg name="nodeTopic" value="at_a_node" />
    <arg name="artifactsTopic" value="artifact_array/relay" />
    <arg name="anchorX" value="-5.0" />
    <arg name="anchorY" value="0.0" />
    <arg name="anchorZ" value="0.0" />

    <arg name="vehicle" value="$(arg vehicle)" />
    <arg name="type" value="$(arg type)" />
    <arg name="sharePoseGraph" value="$(arg sharePoseGraph)" />
    <arg name="sendImages" value="$(arg sendImages)" />
    <arg name="fuseDist" value="$(arg fuseDist)" />
    <arg name="singleGoalDeconflict" value="$(arg singleGoalDeconflict)" />
    <arg name="myBeacons" value="$(arg myBeacons)" />
    <arg name="potentialNeighbors" value="$(arg potentialNeighbors)" />
    <arg name="stopTopic" value="$(arg stopTopic)" />
    <arg name="stopCommand" value="$(arg stopCommand)" />
    <arg name="stopCheck" value="$(arg stopCheck)" />
    <arg name="turnDetect" value="$(arg turnDetect)" />
    <arg name="delayDrop" value="$(arg delayDrop)" />
    <arg name="reverseDrop" value="$(arg reverseDrop)" />
    <arg name="mappingRange" value="$(arg mappingRange)" />
    <arg name="useExtTraj" value="$(arg useExtTraj)" />
  </include>
</launch>
